RELEASE.md

```diff
@@ -50,8 +50,7 @@ This release contains contributions from:
 
 ## Major Features and Improvements
 
-* <INSERT MAJOR FEATURE HERE, USING MARKDOWN SYNTAX>
-* <IF RELEASE CONTAINS MULTIPLE FEATURES FROM SAME AREA, GROUP THEM TOGETHER>
+* Support returning loss history with Scipy optimizer. 
```

Comment:
Add your name below to contributors-list for this release :-)

Reply:
Already there (because of the TF 2.12 support PR) :-)

Reply:
(actually turns out I had an out of date branch, oops; fixed)